### PR TITLE
Adjusted Privacy Policies DE, EN, TR for 2.3

### PR DESCRIPTION
**Pull request by SwissMountainFrog**
This PR updated the privacy policies for app version 2.3

JIRA Ticket https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-6803

### CODE REVIEW

**Review by marcussc — APPROVED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
...
...00|dpvc=1|ssv=4|lastModification=1620627555494|dtVersion=10215210506134511|srmcrl=1|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10215210506134511.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_917400526|rpid=-1422294221|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10207210127152629.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|mel=100000|dpvc=1|lastModification=1613706421428|dtVersion=10207210127152629|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10207210127152629.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-1896203118|rpid=259947634|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10205201218101503.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|dpvc=1|lastModification=1608313592462|dtVersion=10205201218101503|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10205201218101503.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_1499435536|rpid=-978447134|domain=net.sap"></script><style>
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html:8]: I removed the type="text/javascript" intentionally as it creates warning during html validation.

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
...0|dpvc=1|ssv=4|lastModification=1620627555494|dtVersion=10215210506134511|srmcrl=1|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10215210506134511.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-19399945|rpid=407973972|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10207210127152629.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|mel=100000|dpvc=1|lastModification=1613706421428|dtVersion=10207210127152629|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10207210127152629.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-1075122352|rpid=-1376331031|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10205201218101503.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|dpvc=1|lastModification=1608313592462|dtVersion=10205201218101503|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10205201218101503.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-1250168048|rpid=-368485444|domain=net.sap"></script><style>
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html:8]: Same

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/privacy-policy.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
...
...00|dpvc=1|ssv=4|lastModification=1620627555494|dtVersion=10215210506134511|srmcrl=1|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10215210506134511.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-532549878|rpid=-1532543358|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10207210127152629.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|mel=100000|dpvc=1|lastModification=1613706421428|dtVersion=10207210127152629|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10207210127152629.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_572501457|rpid=-1507620304|domain=net.sap"></script><script type="text/javascript" src="/ruxitagentjs_ICA27SVfqrux_10205201218101503.js" data-dtconfig="app=ea7c4b59f27d43eb|featureHash=ICA27SVfqrux|vcv=2|rdnt=1|uxrgce=1|bp=3|cuc=cuve8t91|dpvc=1|lastModification=1608313592462|dtVersion=10205201218101503|tp=500,50,0,1|uxdcw=1500|vs=2|agentUri=/ruxitagentjs_ICA27SVfqrux_10205201218101503.js|reportUrl=/rb_3cbfc0c9-a571-451d-b533-3ebd26e9f5f5|rid=RID_-883486362|rpid=746598318|domain=net.sap"></script><style>
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/tr.lproj/privacy-policy.html:8]: same

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html (hunk 1 above)
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html:8]: I removed the type="text/javascript" intentionally as it creates warning during html validation.

Comment (naveeddotio) [inline src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html:8]: I removed the type="text/javascript" intentionally as it creates warning during html validation.

**Review by naveeddotio — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html (hunk 1 above)
Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html]: I removed it.

File: src/xcode/ENA/ENA/Resources/Localization/tr.lproj/privacy-policy.html (hunk 3 above)
Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Resources/Localization/tr.lproj/privacy-policy.html]: I removed it.

File: src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html (hunk 2 above)
Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Resources/Localization/en.lproj/privacy-policy.html]: I removed it.

**Review by ArturFriesen — APPROVED** [review]

**Pull request merged**